.ui/library/lviHomeCategories.cpx

```diff
@@ -0,0 +1,94 @@
+{
+	"components": [
+		{
+			"className": ".sf-listViewItem .lviHomeCategories",
+			"id": "0892-218d-bc64-b1ba",
+			"props": {
+				"children": [
+					"0595-307b-ea39-8bec"
+				],
+				"name": "lviHomeCategories",
+				"parent": "57f4-201f-4bfc-5fc6"
+			},
+			"source": {
+				"page": "__library__"
+			},
+			"type": "ListViewItem",
+			"userProps": {}
+		},
+		{
+			"className": ".sf-flexLayout",
```

Comment:
Why this flex exists? We should remove it if no special styles used in it.

Reply:
done.